lecture-variables.html

```diff
@@ -63,23 +63,36 @@
     <pre><code>var bensFavLanguage = "JavaScript"
 var studentsFavLanguage = "French"
 console.log("Student's favorite language was " + studentsFavLanguage)
-// It was French!
+// Student's favorite language was French
 
 // .... after this lecture ....
 studentsFavLanguage = bensFavLanguage
 console.log("Student's favorite language was " + studentsFavLanguage)
-// Now it's JavaScript!</code></pre>
+// Student's favorite language was JavaScript</code></pre>
+
+    <p>
+        Notice how in the code snippet above our second <code>console.log</code> statement is identical to the first <code>console.log</code> statement but the output is drastically different since the value of the var <code>studentsFavLanguage</code> has changed.
```

Comment:
Oh @mottaquikarim FYI i am using tabs-not-spaces and "two spaces" as my indent. Make sure to update your editor accordingly so we have consistent indentation